src/+otp/+arenstorf/jac.m

```diff
@@ -0,0 +1,9 @@
+function dfy = jac(~, y, m1 ,m2)
+
+dfy =[
+                                                                                                                                                                                                 0,                                                                                                                                                   0,  1, 0;
+                                                                                                                                                                                                 0,                                                                                                                                                   0,  0, 1;
+ (3*m1*(2*m2 - 2*y(1))*(m2 - y(1)))/(2*((m2 - y(1))^2 + y(2)^2)^(5/2)) - m1/((m2 - y(1))^2 + y(2)^2)^(3/2) - m2/((m1 + y(1))^2 + y(2)^2)^(3/2) + (3*m2*(2*m1 + 2*y(1))*(m1 + y(1)))/(2*((m1 + y(1))^2 + y(2)^2)^(5/2)) + 1,                                                     (3*m2*y(2)*(m1 + y(1)))/((m1 + y(1))^2 + y(2)^2)^(5/2) - (3*m1*y(2)*(m2 - y(1)))/((m2 - y(1))^2 + y(2)^2)^(5/2),  0, 2;
+                                                                                           (3*m2*y(2)*(m1 + y(1)))/((m1 + y(1))^2 + y(2)^2)^(5/2) - (3*m1*y(2)*(2*m2 - 2*y(1)))/(2*((m2 - y(1))^2 + y(2)^2)^(5/2)), (3*m1*y(2)^2)/((m2 - y(1))^2 + y(2)^2)^(5/2) - m1/((m2 - y(1))^2 + y(2)^2)^(3/2) - m2/((m1 + y(1))^2 + y(2)^2)^(3/2) + (3*m2*y(2)^2)/((m1 + y(1))^2 + y(2)^2)^(5/2) + 1, -2, 0];

```

Comment:
The spacing here is weird.  Also potential to move repeated subexpressions into auxiliary variables